plugin.video.mlbtv/addon.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.mlbtv" name="MLB.TV®" version="2016.7.28" provider-name="eracknaphobia">
+  <requires>
+   <import addon="xbmc.python" version="2.19.0"/>   
+   <import addon="script.module.simplejson" version="3.3.0"/>  
+   <import addon="script.module.pytz" version="2014.2" />   
+   <import addon="script.module.beautifulsoup4" version="4.3.1" />
+   <import addon="script.module.pil" version="1.1.7"/>
```

Comment:
I was using it to create the game icons on the fly. Is there a different way to create an image?

def createGameIcon(home,away,image_path):        
    #bg = Image.new('RGB', (400,225), (0,0,0))    
    bg = Image.new('RGB', (500,250), (0,0,0)) 
    #http://mlb.mlb.com/mlb/images/devices/240x240/110.png
    #img_file = urllib.urlopen('http://mlb.mlb.com/mlb/images/devices/76x76/'+home+'.png ')
    img_file = urllib.urlopen('http://mlb.mlb.com/mlb/images/devices/240x240/'+home+'.png ')
    im = StringIO(img_file.read())
    home_image = Image.open(im)
    #bg.paste(home_image, (267,74), home_image)
    bg.paste(home_image, (255,5), home_image)

    #img_file = urllib.urlopen('http://mlb.mlb.com/mlb/images/devices/76x76/'+away+'.png ')
    img_file = urllib.urlopen('http://mlb.mlb.com/mlb/images/devices/240x240/'+away+'.png ')
    im = StringIO(img_file.read())
    away_image = Image.open(im)
    #bg.paste(away_image, (57,74), away_image)    
    bg.paste(away_image, (5,5), away_image)    
    
    bg.save(image_path)